### DISCUSSION TITLE
Offline use, download all  needed files just one time.

### DISCUSSION THREAD

**Discussion by dsienra** [smy20011/dreambooth-gui#83] [General]
The app can be used offline, or is planed on the future.
Why I need to download huge files every time y want to train a different model?
Can be an option to download everything just one time?

Thanks.

**Comment by cheezecrisp** [comment]
Agree. Every time the app downloads a yaml from raw.githubusercontent.com and a 1.71G file from huggingface.co
Maybe there should be a path option to load those files locally.

**Comment by smy20011** [comment]
Wired, It should cache everything. Could you open %APPDIR%/smy20011.dreambooth to make sure files are there?

**Reply by dsienra** [reply]
If I use a local model that I want to retrain downloads every time again a 1.71 GB file.
The app always needs an internet connection to work.

**Reply by dsienra** [reply]
I can confirm that I have a 21gb folder named .hub on %APPDIR%/smy20011.dreambooth
But still re downloads the 1.71 GB file and needs an internet connection to work.

**Comment by smy20011** [comment]
Thank you for reporting this issue. I think I fix this bug in v0.1.11. All the release pipeline should be done within 30 mins.
